Classes/Base.php

```diff
@@ -240,7 +240,7 @@ public static function parseTS( $tsString )
 	 * @param array $typoScriptArray TypoScript configuration array
 	 * @return array Multi-dimensional, associative list of key/value pairs without dots in keys
 	 */
-	protected static function convertTypoScriptArrayToPlainArray(array $typoScriptArray)
+	public static function convertTypoScriptArrayToPlainArray(array $typoScriptArray)
```

Comment:
Great, that would have been a suggestion of me! Have you already changed this in Aimeos\Aimeos\Base?